frontend/src/index.css

```diff
@@ -1,5 +1,20 @@
-body {
-  margin: 0;
-  padding: 0;
-  font-family: sans-serif;
+/* Google Charts */
+
+/* Hide the useless displayZoomButtons */
+#reactgooglegraph-1_AnnotationChart_zoomControlContainer_1-hour,
+#reactgooglegraph-1_AnnotationChart_zoomControlContainer_1-day,
+#reactgooglegraph-1_AnnotationChart_zoomControlContainer_5-days {
+  display: none;
 }
+/* Correct the legend & zoomControls font */
+.google-visualization-atl .legend,
+.google-visualization-atl .zoomControls {
+  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
```

Comment:
I think that if we wrap the chart in a ``Typography`` component, this should be fixed. E.g.:
```patch
diff --git a/frontend/src/pages/AnalyticsPage.js b/frontend/src/pages/AnalyticsPage.js
index e457286..14e3ca2 100644
--- a/frontend/src/pages/AnalyticsPage.js
+++ b/frontend/src/pages/AnalyticsPage.js
@@ -47,11 +47,11 @@ class AnalyticsPage extends Component {
                 Viewing history (views per day)
               </Typography>
             <Grid item xs={12}>{
-              chartData.length > 1 ? <Chart
+              chartData.length > 1 ? <Typography variant='body1' component='div'><Chart
                 chartType="AnnotationChart"
                 data={chartData}
                 options={{fill: 100, colors: ['#EF2E31']}}
-              /> : <Typography variant="subheading">
+              /></Typography> : <Typography variant="subheading">
                 There is no data available for the media
               </Typography>
             }</Grid>
```